spec/models/project_group_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+describe ProjectGroup, :type => :model do
+  it_should_behave_like 'translatable', ['summary', 'details']
```

Comment:
Right now this is pretty much copypasta from ProjectStep. Being less than ideal, I should DRY this up. What would be the preferred method for this? I could just move this to `timeline_entries_spec`, or I could create a `it_should_behave_like 'timeline_entry'`?